src/Http/Headers/src/CookieHeaderValue.cs

```diff
@@ -177,10 +175,7 @@ internal static void CheckNameFormat(StringSegment name, string parameterName)
 
     internal static void CheckValueFormat(StringSegment value, string parameterName)
     {
-        if (value == null)
-        {
-            throw new ArgumentNullException(nameof(value));
-        }
+        ArgumentException.ThrowIfNullOrWhiteSpace(value.Value);
```

Comment:
Behavior change, Why is this more restrictive than before? This might explain most of the auth test failures, an empty value is commonly used when removing a cookie.